python/shotgun_model/data_handler_nav.py

```diff
@@ -146,10 +152,40 @@ class and the data payload from Shotgun should be provided via the
         num_modifications = 0
 
         new_uids = set()
+
+        # a list of sg data dicts to display items for
+        child_data = []
+
+        if item_path == self.__root_path:
+            self._log_debug("This is the root of the tree.")
+            parent_uid = None
+
+            if self.__include_root:
```

Comment:
I'm really worried this will break something. Did you try the search box and expand an asset? How about the home button? Does it take you to the project in the hierarchy?

There's this [bit of code](https://github.com/shotgunsoftware/tk-framework-shotgunutils/blob/develop/zero_config/python/shotgun_model/shotgun_hierarchy_model.py#L124-L140) in the model that assume the path `/Project/<id>` is at the root of model. By having an extra node with the same path somewhere in the tree we might be looking for trouble.